app/components/datepicker/datepicker.js

```diff
@@ -15,20 +15,313 @@
  */
 
 import React from 'react';
+import PropTypes from 'prop-types';
 import { translate } from 'react-i18next';
 import _ from 'lodash';
+import moment from 'moment';
 
-var DATE_FORMAT = 'YYYY-MM-DD';
+function containsAll(str, letters) {
+  const ll = letters.length;
+  let ca = true;
+  for (let i=0; i<ll && ca; i++) {
+    ca = ca && str.indexOf(letters[i]) >= 0;
+  }
+  return ca;
+}
+
+class DatePicker extends React.Component {
```

Comment:
question about this one: is it the one that was used in the patient page when it creates its own account through the web app?

Reply:
Yes